content/docs/apps/managing-teammates.md

```diff
@@ -19,6 +19,22 @@ If you invited them with an agency email address and they're part of an agency t
 
 After a teammate gets a cloud.gov account, an Org Manager or Space Manager will need to give them roles in your orgs and spaces so they can collaborate with you. For details about how org and space roles and permissions work, see [Cloud Foundry roles and permissions](http://docs.cloudfoundry.org/concepts/roles.html#roles).
```

Comment:
Suggestion: `Cloud Foundry roles and permissions` --> `Cloud Foundry's documentation` (perhaps with `about roles and permission`, but I think we don't need it). Why: to give the reader a heads-up that they're about to leave the cloud.gov docs.